src/modules/form/data/adminReferralPosting.json

```diff
@@ -0,0 +1,58 @@
+{
+  "item": [
+    {
+      "type": "CHOICE",
+      "linkId": "denialReason",
+      "required": true,
+      "text": "Reason for Denial",
+      "fieldName": "denialReason",
+      "pickListReference": "REFERRAL_POSTING_DENIAL_REASON_TYPES"
```

Comment:
We should be able to use a "local" picklist for the denial reasons. We only really need to fetch them from the backend if the list is dynamic, or if it's grouped.

The way we'd do that is by resolving denialReson as an Enum. https://github.com/greenriver/hmis-warehouse/pull/3159/files#r1231425655
Once that Enum is in the graphql schema, you can pickListReference it here.